src/riak_core_util.erl

```diff
@@ -150,6 +155,55 @@ replace_file(FN, Data) ->
             Err
     end.
 
+%% @doc Similar to {@link file:consult/1} but will consult files that
+%% are not valid unicode. Specifially files written with
+%% `io_lib:format("~p,", io_list())' in r16 before `utf8' became the
+%% default encoding.
+-spec consult(file:filename()) -> {ok, [term()]} | {error, Err :: term()}.
+consult(FName) ->
+    case file:consult(FName) of
+        {ok, Terms} ->
+            {ok, Terms};
+        {error, _Reason} ->
```

Comment:
I think he means that you could get a POSIX error like `enoent` or `eaccess`, which should be distinguished from a decoding error.